packages/database/src/changeHandlers/entityHierarchyCacher/EntityHierarchyCacher.js

```diff
@@ -60,6 +62,17 @@ export class EntityHierarchyCacher extends ChangeHandler {
     const start = Date.now();
     const subtreeRebuilder = new EntityHierarchySubtreeRebuilder(transactingModels);
     await subtreeRebuilder.rebuildSubtrees(rebuildJobs);
+
+    // explicitly flag ancestor_descendant_relation as changed so that model level caches are cleared
+    transactingModels.database.markRecordsAsChanged(
+      TYPES.ANCESTOR_DESCENDANT_RELATION,
+      rebuildJobs.map(({ hierarchyId, rootEntityId }) => ({
```

Comment:
Decided to send out a change notification per subtree rebuild. Not required, but opens the door for some nice ability to only clear requests from the cache that would have changed (at the moment the whole cache is blown up whenever an entity relation changes)